tabular/src/autogluon/tabular/learner/abstract_learner.py

```diff
@@ -158,7 +158,9 @@ def predict_proba(self, X: DataFrame, model=None, as_pandas=True, as_multiclass=
             X_index = copy.deepcopy(X.index)
         else:
             X_index = None
-        y_pred_proba = self.load_trainer().predict_proba(self.transform_features(X), model=model)
+        y_pred_proba = []
+        if not X.empty:
+            y_pred_proba = self.load_trainer().predict_proba(self.transform_features(X), model=model)
```

Comment:
expects np output if not pandas I think, you might want to check.

probably better to do the 4 lines:

```
if X.empty:
    y_pred_proba = np.ndarray([])
else:
    ...
```

Reply:
Well, but the method signature says X is a dataframe, that's why I used empty()